website/blog/en/2020/pixel-bencmhark.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Running ClickHouse on an Android phone'
+image: 'pixel-benchmark/main.jpg'
```

Comment:
To avoid growing the main repo size there's a separate repo for blog images: https://github.com/ClickHouse/clickhouse-blog-images